src/main/java/org/spongepowered/api/plugin/PluginContainer.java

```diff
@@ -52,6 +54,13 @@
     String getVersion();
 
     /**
+     * Returns the assigned logger to this {@link Plugin}.
+     *
+     * @return The assigned logger
+     */
+    Logger getLogger();
```

Comment:
At the end they will be able to get it anyway using `LoggerFactory.getLogger(container.getId())` so it doesn't really make a difference.